containers/function-handler-java/README.md

```diff
@@ -0,0 +1,28 @@
+# Function handler written in Java
+
+A simple example to run Java Serverless Functions on top of Scaleway Serverless Containers
+
+The example is strongly inspired by this [Flask project written in Python](https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples/container) with PORT definition by Environment variable.
+
+## Setup
+
+- Install and configure [Serverless Framework](https://www.serverless.com/)
+- Install the [Scaleway plugin](https://github.com/scaleway/serverless-scaleway-functions)
+- Install [Your Favorite Java IDE](https://www.jetbrains.com/idea/) with Maven support
```

Comment:
```suggestion
- For local testing install Java and Maven
```

As it's not required to install java and maven + IDE to deploy the container I suggest this change